dist/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
+  <!-- <link rel="stylesheet" href="style.css"> -->
+<script defer src="main.js"></script></head>
+<body class="container">
+  <header>
+    <h1>Leaderboard</h1>
+  </header>
+  <main>
+    <section>
+      <div class ="flex">
+        <h3 class ='mt-3'>Recent scores</h3>
+        <button id ="refresh">Refresh</button>
+      </div>
+      <ul id = 'list'></ul>
+    </section>
+    <section class = flex-column>
+      <h3>Add your score</h3>
+      <input type="text" id ="name" placeholder="Your name">
+      <input type="text" id ="score" placeholder="Your score">
+      <button id ="submit" >Submit</button>
+    </section>
+  </main>
+</body>
```

Comment:
Good job with the `package.json` as well as every other requirements you've implemented. :clap: You've also kept your repository professional as well. :+1: 

## Required changes

- [ ] Please make the layout of your page look like the provided wireframe. Below is what your page currently looks like. Kindly compare it with the provided wireframa and make neccessary adjustments.

![Screenshot 2021-10-05 at 10-17-50 Leaderboard](https://user-images.githubusercontent.com/70150423/136005526-44876550-f37b-457e-80c4-f347821a4042.png)

- [ ] Please clean up the commented code in the `head` tag of your HTML.
- [ ] Pull Request is meant for summary the features you hope to merge in your branch. Here, you have simply repeated (copy/pasted) the project's requirements. Please, let's replace that with the summary of the features you have implemented in your branch.